addons/l10n_dk/data/template/account.tax-dk.csv

```diff
@@ -223,8 +223,10 @@
 "","","","","","","","","","tax","refund","dk_coa_7740","-KM","","",""
 "","","","","","","","","","tax","refund","dk_coa_7700","+MVU","-100","",""
 "tax_keuv0","55","K-EU-V-IngenFradrag","Purchase of goods, acquisition VAT (EU), discretionary deduction","","25.0","percent","purchase","True","base","invoice","","+R-A-V","","Varekøb, erhvervelsesmoms (EU), skønsmæssigt fradrag",""
+"","","","","","","","","","tax","invoice","dk_coa_7740","+KM","","",""
 "","","","","","","","","","tax","invoice","dk_coa_7700","-MVU","-100","",""
 "","","","","","","","","","base","refund","","-R-A-V","","",""
+"","","","","","","","","","tax","refund","dk_coa_7740","-KM","","",""
```

Comment:
you can't have -100% in repartition line without at least a positive line? 

that looks silly. I suppose the constraint should be relaxed :shrug:

Reply:
I checked this tax with OLMA. He confirmed the tax didn't make any sense and I checked with him this modification

Reply:
ok to keep the constraint as now, we'll relax it in case we face a real use case where it is needed